src/Core/src/Handlers/WebView/WebViewHandler.iOS.cs

```diff
@@ -104,8 +104,8 @@ public override Size GetDesiredSize(double widthConstraint, double heightConstra
 
 			var set = false;
 
-			var width = widthConstraint;
-			var height = heightConstraint;
+			var width = size.Width;
```

Comment:
With this changes, the next conditions like:

`if (size.Width == 0)`

can use directly the `width` variable.

Reply:
Thanks for the feedback, I have modified the changes @jsuarezruiz